features/steps/oauth.py

```diff
@@ -4,11 +4,12 @@
 from behave import given, then, when
 import requests
 
-from features.steps import utils
+from features.steps import s4s, utils
 from testsuite.oauth import authorize
 from testsuite import fhir
 
 
+AUTHORIZATION_ACTIONS = ['allow', 'deny']
```

Comment:
I'm using "allow" and "deny" instead of "approve" and "deny" because a) that was the language used in the allscripts `.feature`s and b) "approveing" is not a word.